src/scheduler/job_info.c

```diff
@@ -1475,6 +1479,8 @@ new_job_info()
 	jinfo->attr_updates = NULL;
 	jinfo->resreleased = NULL;
 	jinfo->resreq_rel = NULL;
+	jinfo->depend_job_str = NULL;
+	jinfo->dependent_jobs = NULL;
```

Comment:
I did, since the jobs can only be associated after duping all jobs in server, I call "associate_dependent_jobs(nsinfo)" in dup_server_info to achieve this.